bundles/org.eclipse.cdt.lsp.clangd/.project

```diff
@@ -20,9 +20,15 @@
 			<arguments>
 			</arguments>
 		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.pde.api.tools.apiAnalysisBuilder</name>
+			<arguments>
```

Comment:
BTW, I have an impression that this is the one who slows us down. For Mylyn workspace that has comparable amount of projects and no API Analysis Builder configured I have much better performance for resource delta processing.

Reply:
Yeah, the performance of PDE here can kind of suck :-( I have been using japicmp for LSP4J ([example](https://download.eclipse.org/lsp4j/updates/releases/0.21.0/japicmp-report/)) to do post build analysis which is probably good enough for many uses.

If you the plug-in not API I guess you can remove api analysis builder? However I hope one day that someone fixes [this bug](https://bugs.eclipse.org/bugs/show_bug.cgi?id=570692) which checks the consumer side of API.